nestkernel/recording_device.cpp

```diff
@@ -688,69 +764,91 @@ nest::RecordingDevice::record_event( const Event& event, bool endrecord )
     {
       B_.fs_ << '\n';
       if ( P_.flush_records_ )
+      {
         B_.fs_.flush();
+      }
     }
   }
 
   // storing data when recording to accumulator relies on the fact
   // that multimeter will call us only once per accumulation step
   if ( P_.to_memory_ || P_.to_accumulator_ )
+  {
     store_data_( sender, stamp, offset, weight, target, port, rport );
+  }
 }
 
 void
 nest::RecordingDevice::print_id_( std::ostream& os, index gid )
 {
   if ( P_.withgid_ )
+  {
     os << gid << '\t';
+  }
 }
 
 void
 nest::RecordingDevice::print_time_( std::ostream& os,
   const Time& t,
   double offs )
 {
-  if ( !P_.withtime_ )
+  if ( not P_.withtime_ )
+  {
     return;
+  }
 
   if ( P_.time_in_steps_ )
   {
     os << t.get_steps() << '\t';
     if ( P_.precise_times_ )
+    {
       os << offs << '\t';
+    }
   }
   else if ( P_.precise_times_ )
+  {
     os << t.get_ms() - offs << '\t';
+  }
   else
+  {
     os << t.get_ms() << '\t';
+  }
 }
 
 void
 nest::RecordingDevice::print_weight_( std::ostream& os, double weight )
 {
   if ( P_.withweight_ )
-    os << weight << '\t';
+  {
+    os << weight;
```

Comment:
@mschmidt87 As you already remarked in a comment in #674, the tab is appended to all output so that arbitrary output can be added later. We need to find a better way of handling this to avoid trailing tabs. I suggest to have that discussion in #674.